content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -27,6 +27,9 @@ <h3>Objectives</h3>
 
             <div class="col-md-8">
                 <h3>Kubernetes Deployments</h3>
+
+                <p><strong>Please note that Minikube must be used with the Docker Desktop as the driver for running AMD64 images on ARM architecture.</strong></p>
```

Comment:
nit:
I would suggest to use the Note shortcode.

Consider using
```
{{< note >}}
<p>Minikube must be used with the Docker Desktop as the driver for running AMD64 images on ARM architecture.</p>
{{< /note >}}
```

Reply:
Ack.
I've corrected it into this way.